extensions/amp-ad-network-doubleclick-impl/0.1/test/test-doubleclick-rtc.js

```diff
@@ -312,6 +312,42 @@ describes.realWin('DoubleClick Fast Fetch RTC', {amp: true}, env => {
       Object.keys(macros).forEach(macro => {
         expect(customMacros.ATTR(macro)).to.equal(macros[macro]);
       });
+      return customMacros.ADCID().then(adcid => {
+        expect(adcid).to.not.be.null;
+      });
+    });
+
+    it('should return the same ADCID on multiple calls', () => {
```

Comment:
can we add a test for adcid timeout?  Strongly consider breaking macros into separate functions to ease test coverage.